website/pages/en/seminar.js

```diff
@@ -94,6 +94,8 @@ function Seminar(props) {
               <h4 className='mb-2'><translate>What topics are coming up?</translate></h4>
               <p>On <strong>12 Jan</strong> Parity's Master of Validators, Will Pankiewicz, will discuss network security.</p>
               <p>On <strong>19 Jan</strong> Substrate Core Developer Guillaume Tholliere will discuss FRAME v2.</p>
+              <p>On <strong>26 Jan</strong> Dastan Beksamatov from <a href='https://limechain.tech/'>LimeChain</a> will discuss the design&nbsp;
```

Comment:
My full name is: Dastanbek Samatov. And shortly, just Dastan)

Reply:
Sorry about that - are you okay with me using "Dastan" or do you want to use your full name?